Gems/ScriptCanvas/Code/Builder/ScriptCanvasBuilderWorker.cpp

```diff
@@ -58,12 +59,20 @@ namespace ScriptCanvasBuilder
         else
         {
             AZ_TracePrintf(s_scriptCanvasBuilder, "Failed to load the file: %s", fullPath.c_str());
-            response.m_result = AssetBuilderSDK::CreateJobsResultCode::Failed;
+            return;
         }
 
-        // in terms of job creation, assert on anything but smooth sailing from this point
-        AZ_Assert(sourceGraph, "Graph component is missing from entity.");
-        AZ_Assert(graphData, "GraphData is missing from entity");
+        if (!sourceGraph)
+        {
+            AZ_Error(s_scriptCanvasBuilder, false, "Graph Component missing after successfully loaded: %s", fullPath.c_str());
```

Comment:
No, at this point, this would be corruption of file, bad json editing in an external editor, or something else. I'm just removing asserts pretty aggressively from places where they do not belong.